frontend/app/login/page.js

```
@@ -0,0 +1,85 @@
+                    <h2 className="text-3xl font-bold text-gray-800">Welcome Back!</h2>
+                    <p className="mt-2 text-gray-600">
+                        Log in to access your account and explore the best healthcare services.
+                    </p>
+                </div>
+            }
+            form={
+                <div>
+                    <h2 className="text-2xl font-semibold mb-4 text-gray-800">Login</h2>
+                    <form onSubmit={handleSubmit} className="space-y-4">
+                        <div>
+                            <label className="block text-sm font-medium text-gray-700">Email or Phone</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)